Conclusion - Future outlook (#148)
mlsa-book/MLSA: book/P1C4_survival.qmd

On the other hand, the time an individual stays in hospital is usually determined as the difference between the admission date-time and discharge date-time, which would constitute a continuous time $Y\in \mathbb{R}_{\geq 0}$.

In practice the differences are often blurred as time-measurement will naturally be discretized at some level and precision beyond some resolution is often not of interest (hospital length of stay might be interesting up to days or hours, but not minutes and seconds).

> Almost all software implementations rely on continuous-time theory, and continuous-time methods are therefore typically applied even when the underlying data are technically discrete.

Reviewer: ```suggestion
Many software implementations rely on continuous-time theory, and continuous-time methods are therefore typically applied even when the underlying data are technically discrete.
```

Moreover, discrete-time methods are often applied to continuous time data and vice versa (see @sec-partition-based-reductions for discrete-time methods applied to continuous data).
It is nevertheless important to make the distinction as it informs mathematical treatment and definition of the different quantities introduced below.

::: {.callout-note}

# Handling events at $\tau = 0$

In the theory, continuous time outcomes are defined on the non-negative Reals, $\tau \in \NNReals$, which allows events to occur at $\tau = 0$.
In practice, most model implementations assume time lies on the positive Reals, $\tau \in \PReals$, and therefore cannot accommodate events at $\tau = 0$.

Although relatively uncommon, observations with $\tau = 0$ do arise in real data and must be addressed during preprocessing.
Some strategies include:

1. Removing observations with $t_i = 0$;
2. Replacing $t_i = 0$ with a small positive constant $\epsilon \in \PReals$;
3. Setting $t_i$ to the smallest observed event time greater than zero.

> CLearly if an observation is censored at $\tau = 0$, it provides no information about the survival process and can be reasonably removed.

Reviewer: ```suggestion

```

> The appropriate handling of events at $\tau = 0$ depends on the time scale and application.

Reviewer: ```suggestion
```

> For example, on a scale of days, an observation with $t_i = 0$ may indicate that no meaningful prediction can be made prior to the event.

Reviewer: ```suggestion
```

> In contrast, for coarser time scales such as years, assigning observations to $t_i = 1$ ensures they are still encoded as high-risk.

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
